test/AutoRest.TestServer.Tests/ExtensionsClientNameTest.cs

```diff
@@ -15,5 +16,12 @@ public void ParametersAreRenamed()
             // TODO: Add more tests here
             TypeAsserts.HasParameter(method, "renamedBodyParameter");
         }
+
+        [Test]
+        public void PropertiesAreRenamed()
+        {
+            TypeAsserts.HasProperty(typeof(OriginalSchema), "RenamedProperty");
+            TypeAsserts.HasProperty(typeof(OriginalSchema), "RenamedPropertyString");
```

Comment:
Is this testing that we (currently) re-case the names of properties? Or is this for future renaming configurability?

Reply:
It's testing that m4 applies x-ms-client-name to properties. It didn't before.